app/src/pages/Login.tsx

```diff
@@ -189,7 +190,8 @@ export default function Login() {
     window.localStorage.setItem("emailForSignIn", email);
     setMessage(successMessage);
 
-    await sendSignInEmail(email);
+    // await sendSignInEmail(email);
+    await signInPasswordless(email);
```

Comment:
ah ok so we don't have to catch, it looks ok to just go ahead with this then?

Reply:
I added more try/catch stuff just in case